js/src/sdk/models/triggers.ts

```
@@ -99,67 +119,101 @@ export class Triggers {
-   * @returns {Promise<SingleTriggerRes>} A promise that resolves to the trigger information.
+   * @returns {Promise<TriggerSingleConfig>} A promise that resolves to the trigger configuration.
    * @throws {ComposioError} If the request fails.
    */
-  async getTriggerInfo(data: TriggerSingleParam): Promise<SingleTriggerRes> {
+  async get(data: TriggerSingleParam): Promise<SingleTriggerRes> {
     TELEMETRY_LOGGER.manualTelemetry(TELEMETRY_EVENTS.SDK_METHOD_INVOKED, {
-      method: "getTriggerInfo",
+      method: "get",
+      file: this.fileName,
+      params: { data },
+    });
+    return this.getTriggerInfo(data);
+  }
+
+  /**
+   * @deprecated use trigger.get instead
+   * Retrieves the configuration of a single trigger.
+   *
+   * @param {TriggerSingleParam} data The data for the request.
+   * @returns {Promise<TriggerSingleConfig>} A promise that resolves to the trigger configuration.
+   * @throws {ComposioError} If the request fails.
+   */
+  async getTriggerConfig(
+    data: TriggerSingleParam
+  ): Promise<TriggerSingleConfig> {
+    TELEMETRY_LOGGER.manualTelemetry(TELEMETRY_EVENTS.SDK_METHOD_INVOKED, {
+      method: "getSingleTriggerConfig",
       file: this.fileName,
       params: { data },
     });
     try {
       const parsedData = ZSingleTriggerParam.parse(data);
+
+      if (!parsedData.triggerName && !parsedData.triggerId) {
+        throw CEG.getCustomError(
+          COMPOSIO_SDK_ERROR_CODES.COMMON.INVALID_PARAMS_PASSED,
+          {
+            message: "Trigger name or trigger id is required",
+            description: "Trigger name or trigger id is required",
+            possibleFix: "Pass either triggerName or triggerId",
+          }
+        );
+      }
       const res = await apiClient.triggers.getTriggerInfoV2({
-        path: { triggerName: parsedData.triggerId },
+        path: {
+          triggerName: parsedData.triggerName || parsedData.triggerId || "",
```

The empty string fallback (`|| ""`) could lead to silent failures. Consider throwing an error if both `triggerName` and `triggerId` are falsy values, similar to the validation done in the `getTriggerConfig` method.